pyigm/scripts/pyigm_mtlmcmc.py

```diff
@@ -142,14 +174,16 @@ def run_mcmc_wotta(args):
     ##Test whether to include the carbalpha parameter, and use the correct UVB
     ##...But set a default first
     if not args.grid:
-        args.grid = 'grid_minimal'
+        args.grid = '/afs/crc.nd.edu/group/CGMND/Cloudy_grids/grid_cgm_extensive'
```

Comment:
Let's not hard code this.  Can you have it be 
read in as a parameter?

Reply:
It *is* read in as a parameter (see, e.g., docs/LLS_Metallicities-Wotta/queue_script.sh).  This line just sets a default if it doesn't receive an argument.

Reply:
(see also the documentation at the bottom of the script for input arguments)